config/settings/review_aks.yml

```diff
@@ -0,0 +1,21 @@
+skylight:
+  enable: false
+
+environment:
+  label: "Review"
+  name: "review"
+
+publish_url: https://qa.publish-teacher-training-courses.service.gov.uk
+find_url: https://qa.find-postgraduate-teacher-training.service.gov.uk
```

Comment:
any chance we can use

```
https://publish-review-${APP_NAME}.test.teacherservices.cloud

"find-review-${var.app_suffix}.${var.cluster}.teacherservices.cloud"

```

:pray:

Reply:
@defong this is read by the config gem, looks like you can use ERB: https://github.com/rubyconfig/config#embedded-ruby-erb
As long as APP_NAME, app_suffix, cluster... are available as env vars. This is probably for another card and could be done on paas today.

Reply:
https://trello.com/c/LcOm9e3v

Reply:
@avinhurry Just so you are aware